packages/next/src/build/index.ts

```diff
@@ -656,6 +656,20 @@ async function writeFullyStaticExport(
   appWorker.close()
 }
 
+async function getBuildId(
+  isGenerateMode: boolean,
+  distDir: string,
```

Comment:
Do we need a separate argument for `distDir` or should we derive it from `NextConfigComplete`?

Reply:
distDir is the absolute path, should probably rename the variable in a follow-up PR tbh.